src/mongo_crypt-v1.h

```diff
@@ -23,12 +23,12 @@
 
```

Comment:
Are there changes to this file other than formatting changes? It was copied directly from the upstream code, and I'd prefer to keep it identical to that file so that changes here are clear in the file history. Potentially want to add a `// clang-format off` to the top of this file.

Reply:
Reverted file and added clang-format off to the top.